src/ConstructiveSolidGeometry/VolumePrimitives/Box.jl

```diff
@@ -42,7 +42,7 @@ function Geometry(::Type{T}, ::Type{Box}, dict::AbstractDict, input_units::Named
         _parse_value(T, dict["hY"], length_unit), 
         _parse_value(T, dict["hZ"], length_unit)
     elseif haskey(dict, "x") && haskey(dict, "y") && haskey(dict, "z")
-        @warn "Deprication warning: Detected old primitive definition for `Box`. 
+        @warn "Deprecation warning: Detected old primitive definition for `Box`. 
             Please update your configuration file to the new format 
             via `widhts`, `haldfwidht` or `hX`, `hY` and `hZ`
```

Comment:
I guess this should be `width` and `halfwidth` ?

Reply:
Thanks, I fixed it.